docs/Makefile

```diff
@@ -42,37 +42,44 @@ clean:
 	-rm -rf $(BUILDDIR)/*
 
 html:
+	python3 -c "import sphinx" || python3 -m pip install sphinx
```

Comment:
And when we have multiple we can remove duplication something like:

```suggestion
install-sphinx:
	python3 -c "import sphinx" > /dev/null 2>&1 || python3 -m pip install sphinx

html: install-sphinx
```

Reply:
Ok, done